<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>352</width>
               <height>80</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>352</width>
               <height>80</height>
              </size>
             </property>
             <widget class="QLabel" name="err_tip">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>20</y>
                <width>211</width>
                <height>41</height>
               </rect>
              </property>
              <property name="text">
               <string>错误提示</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="user_label">
               <property name="text">
                <string>用户:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="user_lineEdit">
               <property name="minimumSize">
                <size>
                 <width>280</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>280</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="email_label">
               <property name="text">
                <string>邮箱：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="email_lineEdit">
               <property name="minimumSize">
                <size>
                 <width>280</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>280</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="pass_label">
               <property name="text">
                <string>密码：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="pass_lineEdit">
               <property name="minimumSize">
                <size>
                 <width>280</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>280</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="confirm_label">
               <property name="text">
                <string>确认：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="confirm_lineEdit">
               <property name="minimumSize">
                <size>
                 <width>280</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>280</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="varify_label">
               <property name="text">
                <string>验证码：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="varify_lineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="get_code">
               <property name="text">
                <string>获取</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="confirm_btn">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>确认</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="cancel_btn">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>取消</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
